src/components/criticalErrorMessage.jsx

```diff
@@ -0,0 +1,12 @@
+const newIssue = 'https://github.com/mozilla/firefox-health-dashboard/issues/new';
+
+const CriticalErrorMessage = () => (
+  <p style={{ textAlign: 'center', fontSize: '1.5em' }}>
+    <span>There has been a critical error. We have reported it. If the issue is not fixed
+      within few hours please file an issue:</span>
+    <br />
+    <a href={newIssue} target="_blank">{newIssue}</a>
```

Comment:
You should get in the habit of using `rel="noopener noreferrer"` when using `a` tags to external links for security purposes (tab nabbing).